openpype/modules/base.py

```diff
@@ -367,27 +343,29 @@ def _load_modules():
             # TODO add more logic how to define if folder is module or not
             # - check manifest and content of manifest
             try:
-                if os.path.isdir(fullpath):
-                    # Module without init file can't be used as OpenPype module
-                    #   because the module class could not be imported
-                    init_file = os.path.join(fullpath, "__init__.py")
-                    if not os.path.exists(init_file):
-                        log.info((
-                            "Skipping module directory because of"
-                            " missing \"__init__.py\" file. \"{}\""
-                        ).format(fullpath))
-                        continue
+                # Don't import dynamically current directory modules
+                if is_in_current_dir:
+                    import_str = "openpype.modules.{}".format(basename)
+                    new_import_str = "{}.{}".format(modules_key, basename)
+                    default_module = __import__(import_str, fromlist=("", ))
+                    sys.modules[new_import_str] = default_module
+                    setattr(openpype_modules, basename, default_module)
+
+                elif os.path.isdir(fullpath):
                     import_module_from_dirpath(dirpath, filename, modules_key)
 
-                elif ext in (".py", ):
+                else:
                     module = import_filepath(fullpath)
                     setattr(openpype_modules, basename, module)
 
             except Exception:
-                log.error(
-                    "Failed to import '{}'.".format(fullpath),
-                    exc_info=True
-                )
+                if is_in_current_dir:
+                    msg = "Failed to import default module '{}'.".format(
+                        basename
+                    )
+                else:
+                    msg = "Failed to import '{}'.".format(fullpath)
```

Comment:
Is this clearer if it reads *Failed to import module '{}'*?
```suggestion
                    msg = "Failed to import module '{}'.".format(fullpath)
```